hw3task1.py

```diff
@@ -0,0 +1,14 @@
+mlist = [1, 2, 3, 4, 5, 6, 7, 8]
+even = []
+odd = []
+for index in range(len(mlist)):
+    if index % 2 == 0:
+        even_tuple = ("index - " + str(index) + ", key- " + str(mlist[index]))
+        even.append(even_tuple)
+
+    if index % 2 != 0:
+        odd_tuple = ("index - " + str(index) + ", key-" + str(mlist[index]))
+        odd.append(odd_tuple)
```

Comment:
it works. But over-engineering.
please use for each loop and enumerate.

you do not need a second if. else is okay here